charts/incubator/dendrite/values.yaml

```diff
@@ -0,0 +1,246 @@
+image:
+  repository: ghcr.io/matrix-org/dendrite-monolith
+  pullPolicy: IfNotPresent
+  tag: v0.13.6@sha256:854e156339fcfcf5524290caaec8e2509911bf21e0f0edc7f2d4b25b0cd2eadc
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        port: 8008
+  service:
+    enabled: true
+    ports:
+      service:
+        enabled: true
+        protocol: http
+        port: 8448
+
+workload:
+  main:
+    podSpec:
+      initContainers:
+        init-config:
+          enabled: true
+          imageSelector: image
+          type: init
+          envFrom:
+            - configMapRef:
+                name: dendrite-env
+          command:
+            - /bin/sh
+            - -c
+          args:
```

Comment:
Move this to a configmap, mount it on a temp location, eg
/tmp/tc/config.yaml

have the init container to copy the file to the destination if it does not exist.

maybe use the `yqImage` and update the DB credentials on each startup? (See homeassistant)

Reply:
Ok that makes more sense